rsc/src/main/scala/rsc/scalasig/Pickle.scala

```diff
@@ -1049,17 +1052,23 @@ class Pickle private (settings: Settings, mtab: Mtab, sroot1: String, sroot2: St
     }
     def svalField(sgetterSym: String): s.SymbolInformation = {
       val noGetter = sgetterSym.isPrivateThis && !sgetterSym.isLazy
+      // In traits, a "getter" method is still emitted.
+      val method = sgetterSym.owner.isTrait && noGetter
       val sfieldName = {
         if (noGetter) sgetterSym.desc.value
         else sgetterSym.desc.value + " "
```

Comment:
This is pre-existing code, but do you know what a space is appended if `noGetter` is false?

Reply:
Otherwise the name of the getter will conflict with the name of the field